examples/step-45/step-45.cc

```diff
@@ -574,16 +568,18 @@ namespace Step45
                   for (unsigned int j = 0; j <= i; ++j)
                     {
                       local_matrix(i, j) +=
-                        (symgrad_phi_u[i] * symgrad_phi_u[j] -
-                         div_phi_u[i] * phi_p[j] - phi_p[i] * div_phi_u[j] +
-                         phi_p[i] * phi_p[j]) *
-                        fe_values.JxW(q);
+                        (symgrad_phi_u[i] * symgrad_phi_u[j] // diffusion
+                         - div_phi_u[i] * phi_p[j]           // pressure force
+                         - phi_p[i] * div_phi_u[j]           // divergence
+                         + phi_p[i] * phi_p[j])              // pressure mass
+                        * fe_values.JxW(q);
```

Comment:
Since step-22 has been recently switched to put the pressure mass matrix into a separate local matrix #6221, would you mind making this adaptation now? I know it is off-topic, but since we're around already I think that would make sense now.